server/services/licence/licenceStatusTypes.ts

```diff
@@ -54,6 +54,7 @@ export type Decisions = {
   dmRefused?: boolean
   refusalReason?: string
   decisionComments?: string
+  dmNotConsidered?: boolean
```

Comment:
I think the naming of `dmNotConsidered` is a bit confusing as if a user answers No they have not considered its true and Yes is false. Maybe makes more sense to call it `dmConsidered` so a truthy value is yes? I know we're flagging that something has not been done but in terms of setting out the logic in code think it's easier to read as `dmConsidered` 🤔

Reply:
Hope I'm understanding correctly 😅 have confused myself

Reply:
Understand the nature of this now and it will stay as is 😄